### ISSUE TITLE
Passive node on Windows 10 seems to sit idle for hours between block updates (0.5.6)

### ISSUE THREAD

**Issue by JSCSJSCS** [input-output-hk/jormungandr#906]
**Mandatory Information**
Operating System: Windows 10 Home build 1903 64bit
Intel I7-9750H CPU @2.6GHz, 16GB RAM
```
jcli 0.5.6 (HEAD-7ab929ea+, release, windows [x86_64]) - [rustc 1.38.0 (625451e37 2019-09-23)]
jormungandr 0.5.6 (HEAD-7ab929ea+, release, windows [x86_64]) - [rustc 1.38.0 (625451e37 2019-09-23)]
```

**Describe the bug**
After starting a passive node in Powershell with the command:
`./jormungandr --config ./node-config.yaml --genesis-block-hash adbdd5ede31637f6c9bad5c271eec0bc3d0cb9efb86a5b913bb55cba549d0770`,
the Powershell output first shows "initial bootstrap completed" and other initial setup info notices, then shows no activity, sometimes for hours. 

```
PS D:\jormungandr-node> ./jormungandr --config ./node-config.yaml --genesis-block-hash adbdd5ede31637f6c9bad5c271eec0bc3d0cb9efb86a5b913bb55cba549d0770
Oct 07 11:31:58.453 INFO Starting jormungandr 0.5.6 (HEAD-7ab929ea+, release, windows [x86_64]) - [rustc 1.38.0 (625451e37 2019-09-23)], task: init
Oct 07 11:31:58.453 WARN Node started without path to the stored secret keys (not a stake pool or a BFT leader), task: init
Oct 07 11:31:58.453 INFO storing blockchain in '"D:/jormungandr-node/storage/blocks.sqlite"', task: init
Oct 07 11:31:58.719 INFO fetching block adbdd5ede31637f6c9bad5c271eec0bc3d0cb9efb86a5b913bb55cba549d0770, peer_address: 3.115.57.216:3000, block: adbdd5ede31637f6c9bad5c271eec0bc3d0cb9efb86a5b913bb55cba549d0770, task: init
Oct 07 11:31:59.222 INFO initial bootstrap completed, peer_address: 3.115.57.216:3000, block: adbdd5ede31637f6c9bad5c271eec0bc3d0cb9efb86a5b913bb55cba549d0770, task: init
Oct 07 11:31:59.347 INFO connecting to bootstrap peer 3.115.57.216:3000, peer_addr: 3.115.57.216:3000, task: bootstrap
Oct 07 11:51:27.288 WARN with initial bootstrap, reason: bootstrap pull stream failed, peer_addr: 3.115.57.216:3000, task: bootstrap
Oct 07 11:51:27.302 INFO connecting to bootstrap peer 18.140.134.230:3000, peer_addr: 18.140.134.230:3000, task: bootstrap
Oct 07 11:56:09.247 WARN with initial bootstrap, reason: bootstrap pull stream failed, peer_addr: 18.140.134.230:3000, task: bootstrap
Oct 07 11:56:09.251 INFO connecting to bootstrap peer 52.57.157.167:3000, peer_addr: 52.57.157.167:3000, task: bootstrap
Oct 07 11:58:34.190 WARN with initial bootstrap, reason: bootstrap pull stream failed, peer_addr: 52.57.157.167:3000, task: bootstrap
Oct 07 11:58:34.193 INFO connecting to bootstrap peer 3.112.185.217:3000, peer_addr: 3.112.185.217:3000, task: bootstrap
Oct 07 12:09:31.230 INFO error connecting to peer, reason: No connection could be made because the target machine actively refused it. (os error 10061), peer_addr: 3.123.155.47:3000, task: network
Oct 07 12:09:41.328 INFO error connecting to peer, reason: No connection could be made because the target machine actively refused it. (os error 10061), peer_addr: 3.123.155.47:3000, task: network
Oct 07 12:09:41.335 INFO client P2P connection closed, node_id: 4485365775303479996, peer_addr: 3.123.155.47:3000, task: network
```

Then there would be pages and pages of this kind of output: Note: turning windows firewall protection off, and/or allowing the node-config.yaml ports through did not stop the below alerts from continuing.

```
Oct 07 12:31:21.274 INFO client P2P connection closed, node_id: 3315279906951703745, peer_addr: 51.91.96.237:2999, task: network
Oct 07 12:31:21.572 INFO error connecting to peer, reason: No connection could be made because the target machine actively refused it. (os error 10061), peer_addr: 188.134.8.225:3000, task: network
Oct 07 12:31:21.576 INFO client P2P connection closed, node_id: 17559116024830897001, peer_addr: 188.134.8.225:3000, task: network
Oct 07 12:31:29.457 INFO error connecting to peer, reason: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (os error 10060), peer_addr: 207.148.73.212:3000, task: network
Oct 07 12:31:29.466 INFO client P2P connection closed, node_id: 14786065015321395650, peer_addr: 207.148.73.212:3000, task: network
Oct 07 12:31:31.282 INFO error connecting to peer, reason: No connection could be made because the target machine actively refused it. (os error 10061), peer_addr: 51.91.96.237:2999, task: network
Oct 07 12:31:31.292 INFO client P2P connection closed, node_id: 3315279906951703745, peer_addr: 51.91.96.237:2999, task: network
Oct 07 12:31:39.463 INFO error connecting to peer, reason: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (os error 10060), peer_addr: 207.148.73.212:3000, task: network
Oct 07 12:31:39.468 INFO client P2P connection closed, node_id: 14786065015321395650, peer_addr: 207.148.73.212:3000, task: network
Oct 07 12:31:41.278 INFO error connecting to peer, reason: No connection could be made because the target machine actively refused it. (os error 10061), peer_addr: 51.91.96.237:2999, task: network
Oct 07 12:31:41.282 INFO client P2P connection closed, node_id: 3315279906951703745, peer_addr: 51.91.96.237:2999, task: network
Oct 07 12:31:49.443 INFO error connecting to peer, reason: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (os error 10060), peer_addr: 3.19.243.190:3333, task: network
Oct 07 12:31:49.451 INFO client P2P connection closed, node_id: 2439400218527819787, peer_addr: 3.19.243.190:3333, task: network
Oct 07 12:31:49.458 INFO error connecting to peer, reason: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (os error 10060), peer_addr: 207.148.73.212:3000, task: network
Oct 07 12:31:49.463 INFO client P2P connection closed, node_id: 14786065015321395650, peer_addr: 207.148.73.212:3000, task: network
Oct 07 12:31:51.287 INFO error connecting to peer, reason: No connection could be made because the target machine actively refused it. (os error 10061), peer_addr: 51.91.96.237:2999, task: network
```

Then, seemingly at random times, there is a burst of P2P INFO notices and Block applied notices.
This behavior was also observed in 0.5.5 and there was hope 0.5.6 would solve it. It does not. Notice it took about 1:45 after starting the node that I got a "block blast" of hundreds of block alerts at the same minute.

```
Oct 07 14:16:37.624 INFO error connecting to peer, reason: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (os error 10060), peer_addr: 3.19.243.190:3333, task: network
Oct 07 14:16:37.629 INFO client P2P connection closed, node_id: 2439400218527819787, peer_addr: 3.19.243.190:3333, task: network
Oct 07 14:16:37.632 INFO block successfully applied, date: 227.23952, parent: 728aad83a7bab909ff73bc49713467b173d84e0640b4b1c108bcdb426285866f, hash: 118743fc112412a35638869cb415620bf7de6d171fb3e0662819a6efaff20f36, task: block
Oct 07 14:16:37.634 INFO update current branch tip, date: 227.23952, parent: 728aad83a7bab909ff73bc49713467b173d84e0640b4b1c108bcdb426285866f, hash: 118743fc112412a35638869cb415620bf7de6d171fb3e0662819a6efaff20f36, task: block
Oct 07 14:16:37.639 INFO error connecting to peer, reason: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (os error 10060), peer_addr: 3.19.243.190:3333, task: network
Oct 07 14:16:37.642 INFO error connecting to peer, reason: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (os error 10060), peer_addr: 136.27.45.31:3100, task: network
Oct 07 14:16:37.645 INFO error connecting to peer, reason: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (os error 10060), peer_addr: 178.27.162.85:80, task: network
Oct 07 14:16:37.650 INFO client P2P connection closed, node_id: 2439400218527819787, peer_addr: 3.19.243.190:3333, task: network
Oct 07 14:16:37.653 INFO client P2P connection closed, node_id: 14382243596378040576, peer_addr: 136.27.45.31:3100, task: network
Oct 07 14:16:37.655 INFO client P2P connection closed, node_id: 536998608617073920, peer_addr: 178.27.162.85:80, task: network
Oct 07 14:16:37.660 INFO error connecting to peer, reason: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (os error 10060), peer_addr: 104.35.47.124:3100, task: network
Oct 07 14:16:37.664 INFO client P2P connection closed, node_id: 11790324710824787456, peer_addr: 104.35.47.124:3100, task: network
Oct 07 14:16:37.666 INFO incoming P2P connection closed, peer_addr: 140.82.56.80:45022, task: network
```

**To Reproduce**
1. Open Powershell and execute the command 
`./jormungandr --config ./node-config.yaml --genesis-block-hash adbdd5ede31637f6c9bad5c271eec0bc3d0cb9efb86a5b913bb55cba549d0770`

**Expected behavior**
I expected that after a short time (10-15 minutes), the output would see P2P and block applied messages as they occurred, rather than all at once in a "Block blast".

**Additional context**
node-config.yaml (port 4000 is forwarded to my public IP Address and tests as open after the listening is started in jormungandr.)

```
storage: "D:/jormungandr-node/storage/"
log:
  format: plain
  level: info
  output: stderr
p2p:
  listen_address: /ip4/0.0.0.0/tcp/4000
  public_address: /ip4/my.ip.ad.rs/tcp/4000
  topics_of_interest:
    blocks: high
    messages: high
  trusted_peers: [/ip4/3.123.177.192/tcp/3000,
                  /ip4/52.57.157.167/tcp/3000,
                  /ip4/3.112.185.217/tcp/3000,
                  /ip4/18.140.134.230/tcp/3000,
                  /ip4/18.139.40.4/tcp/3000,
                  /ip4/3.123.155.47/tcp/3000,
                  /ip4/3.115.57.216/tcp/3000]
rest:
  listen: "127.0.0.1:3101"
```

**Comment by rdlrt** [comment]
You should be using https://github.com/input-output-hk/shelley-testnet repo to report issues.

**Comment by carl-IOHK** [comment]
Actually we have moved to using Zendesk to repport issues for Shelley Networked Testnet.

In order to get a faster response to your question, it's better if you post your questions to the Technical Support Team here:

https://iohk.zendesk.com/hc/en-us/requests/new

**Comment by JSCSJSCS** [comment]
Good to know about the new zendesk issue reporting.

**Comment by NicolasDP** [comment]
duplicate of #860 we are working on this and we hope to have an update soon to improve this.
